<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Candidatures</class>
 <widget class="QDialog" name="Candidatures">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1990</width>
    <height>869</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>561</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>279</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditStatus">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>366</y>
     <width>561</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>171</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Candidat</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Date de candidature</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>376</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>396</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>de la candidature</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>200</y>
     <width>921</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Ajouter_candidature">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>800</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>520</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Offre d'emploi</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>520</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>550</y>
     <width>561</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <zorder>textEditStatus</zorder>
  <zorder>dateEdit</zorder>
  <zorder>listView</zorder>
  <zorder>lineEdit</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>tableView</zorder>
  <zorder>Ajouter_candidature</zorder>
  <zorder>label_5</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>listView_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
